cfn-resources/test/e2e/utility/aws_helper.go

```diff
@@ -179,3 +197,12 @@ func TestIsTemplateValid(t *testing.T, svc *cfn.Client, template string) {
 	_, err := svc.ValidateTemplate(context.Background(), input)
 	FailNowIfError(t, "invalid cloudformation stack: %v", err)
 }
+
+func DeleteStackIfExists(t *testing.T, c *cfn.Client, stackName string) error {
+	_, err := DescribeStack(c, stackName)
+	if err != nil && err.Error() != StackNotFoundErr {
```

Comment:
this is a bad practice in go, do you know if the error for `StackNotFoundErr` can be compared using `errrors.Is`?

Reply:
not required anymore